### ISSUE TITLE
Look into custom exception handler for I18n translate

### ISSUE THREAD

**Issue by Cruikshanks** [DEFRA/ruby-services-team#74]
In production any locale issue such as a missing translation is quietly swallowed, with Rails falling back to outputting a 'human' version of the locale key.

We want this behaviour because we don't want users to be blocked from continuing just because of a missing translation. Also its often the case that the stand-in is close enough that its doesn't look out of place.

In non-production environments we have set Rails to error because we want to know about them and this is one way to spot them.

@andrewhick is also doing what he can through our acceptance tests to check validations work and the errors are as they should be. However we are left with us on the dev/qa team spotting them as we work with the app.

I'd like to explore how we can use a custom exception handler for any I18n issues

- https://guides.rubyonrails.org/i18n.html#using-different-exception-handlers
- https://makandracards.com/makandra/17967-raise-when-there-s-a-i18n-translation-missing
- https://stackoverflow.com/a/9507750

My idea is that by using a custom exception handler we can have it make a call to Airbrake with details of the missing translation, but then allow Rails to behave as it normally would.

My thinking is that if and when our users come across missing translations in the app, we'll now be able to see some notification that they exist, but they won't be blocked from continuing.

> TL;DR Trying to automate the checking for missing translations in a minimal, reliable and maintainable way is I think beyond our time and resources, but at least having a way to know about them if and when users come across them would still have value, and should be in our scope to implement
